libfive/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ if (NOT DEFINED LIBFIVE_BUILD_FROM_ROOT)
     message(FATAL_ERROR "CMake must be called against top-level directory")
 endif()
 
-option(BUILD_TESTS "Build test suite" ON)
+option(BUILD_TESTS "Build test suite" OFF)
```

Comment:
is this to speed up ntop build time? if so, how much time does it save?

Reply:
I dont know, I just noticed it gets build unnecessarily.

Reply:
The idea was to build AND run them. There might be value in doing so. But yeah, just building them is not that useful.

Notice also that this will only affect "Build Solution" which I wouldn't recommend anyway over "Build Project".

Reply:
Ok, I see, then I'll close this for now! IMO there is little value to build this by default for devs since libfive is almost never touched (Otoh it basically never gets rebuild).